documentation/prerequisites.md

```diff
@@ -35,6 +36,7 @@ Following table summarizes key features introduced in vSphere Cloud Provider in
     - With kuberneters version 1.9.x if user wants to use multiple vCenters then vSAN storage can not be used. vSAN is limited to one cluster in one vCenter deployment.
 * vCenter user with required set of privileges.
 * VMware Tools needs to be installed on the guest operating system on each Node VM. Please refer this [link](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.html.hostclient.doc/GUID-ED3ECA21-5763-4919-8947-A819A17980FB.html) for instruction on installing VMware tools.
+* Virtual Hardware Version (VM version) 11 is required for VMs serving as Container Hosts.
```

Comment:
I would recommend explicitly calling out what the hardware version should not be, to avoid any interpretation like- 11 or higher.

Reply:
Can we say  - "Node VMs should have Compatibility "ESXi 6.0 and later (VM version 11)".
We have not tested with any other VM compatibility yet.

Reply:
@ divyenpatel`ESXi 6.0 and later ` implies ESX 6.5 too which by default comes with VM version 13. I think we can clearly only `Virtual Hardware Version (VM version)`
@prashima we have not tested VM version below 11 too.  Anything higher than 11 isn't compatible. So I think we should particularly call out only version 11 . We need further testing to specify if anything below/above 11 is supported as well.